os/drivers/testcase/kernel_testcase_drv.c

```diff
@@ -142,6 +142,7 @@ static int kernel_test_drv_ioctl(FAR struct file *filep, int cmd, unsigned long
 	}
 	break;
 
+#ifdef CONFIG_TC_KERNEL_ROUNDROBIN
```

Comment:
Is this already existing in Kconfig? if so which?

Reply:
https://github.com/Samsung/TizenRT/blob/master/apps/examples/testcase/le_tc/kernel/Kconfig
Here it is.